src/sentry/lang/native/symbolicator.py

```diff
@@ -154,7 +154,10 @@ def _process(self, create_task):
 
             metrics.incr(
                 "events.symbolicator.response",
-                tags={"response": json_response.get("status") or "null"},
+                tags={
+                    "response": json_response.get("status") or "null",
+                    "task_type": getattr(create_task, "__name__", None) or "null",
```

Comment:
To confirm, will it get a proper name if `create_task` is a lambda?

Reply:
you're totally right. I looked at the wrong piece of code. Now I tested locally to make sure